### ISSUE TITLE
Security Scan Doesn't Catch Log Injection Vulnerability

### ISSUE THREAD

**Issue by michaelhoffmantech** [aws/aws-toolkit-jetbrains#3859]
**Describe the bug**
When I run a security scan against a Java application that includes a log injection vulnerability (https://docs.aws.amazon.com/codeguru/detector-library/java/log-injection/), the security scan comes back with no issues, not identifying the log injection vulnerability in the code. 

**To reproduce**
1. Have the latest AWS Toolkit installed and a valid connection with an AWS Builder ID
2. Create the following code with a log injection vulnerability:
```
    @RolesAllowed("ADMIN")
    @JsonView(ConferenceViews.SponsorProspectTopLevel.class)
    @GetMapping("/conference/{conferenceId}/")
    public ResponseEntity<List<SponsorshipProspect>> getSponsorProspectsByConferenceId(@PathVariable Long conferenceId, HttpServletRequest req) {
        String param = req.getParameter("param");
        log.info("param: {}", param);
        return new ResponseEntity<>(this.sponsorshipProspectService.getSponsorshipProspectsByConferenceId(conferenceId), HttpStatus.OK);
    }
```
3. Clicked on Run Security Scan from the Developer Tools AWS Toolkit pane
4. Results come back with Security Scan completed for XXX files. No issues found in "project name". 

**Expected behavior**
I expected the file with the log injection vulnerability to be included as an issue. 

**Screenshots**

**Your Environment**
- OS: Windows 11 Pro, Version 10.0.22621 Build 22621
- JetBrains product: IntelliJ Idea Ultimate Edition
- JetBrains product version:  2023.2.1 Build #IU-232.9559.62, built on August 23, 2023
- AWS Toolkit version: 1.78-232
- SAM CLI version: 
- JVM/Python version: 17.0.2

**Additional context**

**Comment by andrewyuq** [comment]
Hi @michaelhoffmantech,

Does it still not show anything if you directly run security scan with this file actively opened as the current editor?

**Comment by michaelhoffmantech** [comment]
Yes, I've always tried with the file open that has the vulnerability. Let me know if it would help to dump the IntelliJ trace log from execution.
